spec/unit/resource_controller_spec.rb

```diff
@@ -157,7 +157,7 @@ def call_after_destroy(obj); end
 
 describe Admin::PostsController, type: "controller" do
 
-  describe 'retreiving the resource' do
+  describe 'retrieving the resource' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.